polaris-core/src/main/java/org/apache/polaris/core/config/BehaviorChangeConfiguration.java

```diff
@@ -37,9 +37,10 @@ protected BehaviorChangeConfiguration(
     super(key, description, defaultValue, catalogConfig);
   }
 
+  @BehaviorChange(since = "1.0.0")
   public static final BehaviorChangeConfiguration<Boolean> VALIDATE_VIEW_LOCATION_OVERLAP =
       PolarisConfiguration.<Boolean>builder()
-          .key("STORAGE_CREDENTIAL_CACHE_DURATION_SECONDS")
+          .key("VALIDATE_VIEW_LOCATION_OVERLAP")
```

Comment:
WDYT about making this a small separate PR, which this one is still in review?

Reply:
This is just a toy config that I added when introducing the configs, so I don't feel a tremendous sense of urgency to change this. That said, I have opened #1201 in case we want to go ahead and fix this right away. Thanks for the suggestion!

Reply:
no worries. I thought it was a real bug :)